js_modules/dagster-ui/packages/ui-core/src/runs/RunMetadataProvider.tsx

```diff
@@ -460,7 +460,13 @@ export const RUN_METADATA_PROVIDER_MESSAGE_FRAGMENT = gql`
       pid
       externalStdoutUrl
       externalStderrUrl
-      logManagerMetadata
+      logManagerMetadata {
+        ... on LogManagerMetadata {
```

Comment:
shouldn't need to use a `... on Type`  the type here if there only one possible return type